### PR TITLE
feat: US138561 Add tests for d2l-questions-multiple-choice

### PR DESCRIPTION
**Pull request by tdevos-d2l**
[US138561](https://rally1.rallydev.com/#/?detail=/userstory/632747458499&fdp=true): [Consistent Evaluation][Quizzes][UI][Questions] Add tests for d2l-questions-multiple-choice

- Adding testing to the `d2l-questions-multiple-choice` component (Tristan)
- Adding Axe tests to multiple choice presentational (Scy)
- Making v-diff tests test the presentational component directly (Scy)
- Merged skeleton v-diff tests into regular v-diff tests (Tristan)
- Tests for `d2l-questions-question` redone (Scy)

### CODE REVIEW

File: test/d2l-questions-multiple-choice-presentational.axe.test.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { choicesCorrect, choicesIncorrect, choicesUnanswered } from './d2l-questions-multiple-choice-presentational-data.js';
+import { expect, fixture, html } from '@open-wc/testing';
+
+describe('multiple-choice-presentational-axe', () => {
+
+	it('answered', async() => {
```
Comment (maddiegabriel) [inline test/d2l-questions-multiple-choice-presentational.axe.test.js:6]: should we have one for `answered - correct` and `answered - incorrect`?

**Review by maddiegabriel — APPROVED** [review]
the new tests seem robust and well organized - LGTM! 👌

File: test/d2l-questions-multiple-choice-presentational.axe.test.js (hunk 1 above)
Comment (tdevos-d2l) [reply test/d2l-questions-multiple-choice-presentational.axe.test.js:6]: I think only `Readonly` can be incorrect or correct. We don't show the correctness until it's submitted and readonly.
@Scy-D2L could confirm?

Comment (tdevos-d2l) [reply test/d2l-questions-multiple-choice-presentational.axe.test.js:6]: I see we use `.choices=${choicesCorrect}` here and that's confusing. We're just using the question data from the `choicesCorrect` object, not to tell if it's correct or not.

Comment (Scy-D2L) [reply test/d2l-questions-multiple-choice-presentational.axe.test.js:6]: At the moment `readonly` displays correctness and the default 'answerable' state does not. Tristan is correct that it's ignoring the correctness information and only displaying the options / selection.

**Review by JamieTran — APPROVED** [review]
💯

**Pull request merged**